compiler/src/ammtoaga/index.ts

```diff
@@ -10,18 +10,15 @@ type AddressMap = {
 type EventDecs = {
   [name: string]: number;
 };
-type MemoryMap = {
-  [name: string]: number;
-};
 type HandlerMem = {
   memSize: number;
-  addressMap: MemoryMap;
+  addressMap: AddressMap;
```

Comment:
Some of the code referenced addresses as JS numbers, others as BigInt. Since the address space *is* 64-bit just use BigInt everywhere, which simplifies the code a lot.